spec/src/main/asciidoc/bulkhead.asciidoc

```diff
@@ -71,8 +71,21 @@ public Future<Connection> serviceA() {
 
 When using the thread pool approach, when a request cannot be added to the waiting queue, `BulkheadException` will be thrown.
 
+==== Lifecycle
+
+Bulkhead needs to needs to maintain some state between invocations: the number of currently running executions, or the queue of waiting executions.
+This state is a singleton, irrespective of the lifecycle of the bean that uses the `@Bulkhead` annotation.
+In other words, the state is associated with the bean class, not with its instances.
+For example, if the bean is `@RequestScoped` and therefore there is a new instance of the bean for each request, all instances share the same bulkhead state.
+
+More specifically: the bulkhead state is bound to the bean class _and_ the guarded method.
+If all the bulkheads were stored in a `Map`, then the key to the `Map` would be a combination of the bean class (`java.lang.Class`) and the method object (`java.lang.reflect.Method`).
```

Comment:
I would reword to be:
The combination of the bean class (`java.lang.Class`) and the method object (`java.lang.reflect.Method`) uniquely identify a bulkhead.

Reply:
Done.